clippy_utils/src/consts.rs

```diff
@@ -1,3 +1,8 @@
+//! A simple const eval API, for use on arbitrary hir expressions.
+//!
+//! This cannot use rustc's const eval, aka miri, as arbitrary hir expressions cannot be lowered to
+//! executable MIR bodies, so we have to do this instead. If hir expressions gain the ability to be
```

Comment:
Might be use HIR and MIR lower case or upper case everywhere

Reply:
Done, all uppercase now.